src/engines/nwn2/roster.cpp

```diff
@@ -82,11 +105,25 @@ void Roster::load() {
 void Roster::loadMember(const Aurora::GFF3Struct &gff) {
 	Member member;
 
+	initMember(&member);
+
 	// Add a roster member to the list
-	member.rosterName = gff.getString("RosName");
+	member.rosterName = gff.getString("RosName", member.rosterName);
+	member.isAvailable = gff.getBool("RosAvailable", member.isAvailable);
+	member.isCampaignNPC = gff.getBool("RosCampaignNPC", member.isCampaignNPC);
+	member.isSelectable = gff.getBool("RosSelectable", member.isSelectable);
+	member.isLoadBefore = gff.getBool("RosLoadBefore", member.isLoadBefore);
 	_members.push_back(member);
 }
 
+void Roster::initMember(Member *member) {
```

Comment:
Make it take a `Member &` instead, please.

Or you could even add a constructor to struct Member instead.

Reply:
Changed to a constructor.